utils/conversions.ts

```diff
@@ -116,9 +117,17 @@ export function retrievedServiceToVisibleService(
     serviceId,
     serviceMetadata,
     serviceName,
-    version
+    version,
+    _etag,
+    _rid,
+    _self,
+    _ts
```

Comment:
```suggestion
    version
```

Reply:
this function exists to strip these internal fields :)

Reply:
yes, it should only contain the version field but not the internal cosmosdb fields

Reply:
originally it doesn't indeed: https://github.com/pagopa/io-functions-commons/blob/v10.7.0/src/models/visible_service.ts#L33

Reply:
PR opened: [https://github.com/pagopa/io-functions-commons/pull/89](https://github.com/pagopa/io-functions-commons/pull/89)

Reply:
fixed in [#e6e2575](https://github.com/pagopa/io-functions-admin/pull/65/commits/e6e2575ed93102aa692f1732c0e859b710b57a7d)